### PR TITLE
r generator

### PR DESCRIPTION
**Pull request by kasptom**

### CODE REVIEW

**Comment by kasptom** [comment]
[RFactory](https://trello.com/c/TD1boe88/5-rfactory)

File: src/main/java/pl/edu/agh/propertree/finder/ResourceFinder.java [hunk 1]
```diff
@@ -55,6 +55,33 @@ private static Address findWithPrefixOrDefault(int resourceId, String configPref
...
+        return Integer.valueOf(stringValue);
+    }
+
+    private static String findString(int resourceId, String configPrefix) {
+        Address address = findWithPrefixOrDefault(resourceId, configPrefix);
+        return String.valueOf(getValue(address));
+    }
+
     private static String getConfigPrefix(String path) {
         Matcher matcher = POSTFIX_PATTERN.matcher(path);
```
Comment (FreexD) [inline src/main/java/pl/edu/agh/propertree/finder/ResourceFinder.java]: prefix or postfix?

File: src/main/java/pl/edu/agh/propertree/generator/AddressGenerator.java [hunk 2]
```diff
@@ -0,0 +1,233 @@
...
+            hasDouble = false;
+        }
+        return hasDouble;
+    }
+
+    private static boolean hasBoolean(String value) {
+        boolean hasInteger = true;
+        try {
+            //noinspection ResultOfMethodCallIgnored
+            Integer.parseInt(value);
```
Comment (FreexD) [inline src/main/java/pl/edu/agh/propertree/generator/AddressGenerator.java]: int or boolean?

File: src/main/java/pl/edu/agh/propertree/generator/RGenerator.java [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+import javax.lang.model.element.Modifier;
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Class which generates RHardcoded.java on basing on the configuration files
```
Comment (FreexD) [inline src/main/java/pl/edu/agh/propertree/generator/RGenerator.java]: RHardcoded or R?

**Review by FreexD — CHANGES_REQUESTED** [review]

**Review by FreexD — APPROVED** [review]

**Comment by FreexD** [comment]
LGTM

**Pull request merged**